src/activity_handlers.cpp

```diff
@@ -2911,6 +2911,9 @@ void activity_handlers::dig_finish( player_activity *act, player *p )
         }
         g->u.add_memorial_log( pgettext( "memorial_male", "Exhumed a grave." ),
                                pgettext( "memorial_female", "Exhumed a grave." ) );
+        g->m.ter_set( pos, t_pit_shallow );
+    } else {
+        g->m.ter_set( pos, t_pit_shallow );   
```

Comment:
If it's no matter whether `grave` is true or false, `g->m.ter_set( pos, t_pit_shallow );` could be set only once after the `if( grave )` check.

Reply:
Thanks for finding that. That's a typo I missed when resolving merge conflicts. T_pit is the right one. My reasoning behind this is that grave is deeper then a shallow pit, but since the casket takes some place the digging cost is equal to shallow pit.